### PR TITLE
Offer to scrape again if no roms found

### LINKED ISSUE #6: Empty page when no roms found
Add some feedbackk to the user that no roms were found

add link to rescan

### PR DESCRIPTION
**Pull request by hola-soy-milk**
Closes #6

### CODE REVIEW

File: renderer/pages/[folder]/[path]/index.tsx [hunk 1]
```diff
@@ -20,9 +15,29 @@ export const Files = () => {
     folders[query.folder as string]?.folders[query.path as string];
 
   const scrape = () => {
-    scrapeFolder(activeFolder, folders[query.folder as string]);
+    scrapeFolder(activeFolder, folders[query.folder as string], true);
```
Comment (SaraVieira) [inline renderer/pages/[folder]/[path]/index.tsx]: This may have been me but what is this true for?

Comment (hola-soy-milk) [reply renderer/pages/[folder]/[path]/index.tsx]: [This is the `all` boolean](https://github.com/SaraVieira/retromatch/blob/995ae89ce88fa89d9a35c3495886c4e96288c73f/renderer/hooks/folder-context.tsx#L11), which I think scrapes all the contents in the folder if I understand correctly!

Comment (hola-soy-milk) [reply renderer/pages/[folder]/[path]/index.tsx]: I think this `scrape` function wasn't being used and so the signature wasn't up to date

**Comment by SaraVieira** [comment]
pH no merge conflicts :(

**Comment by hola-soy-milk** [comment]
Good catch! I'll clean this up

**Comment by SaraVieira** [comment]
I am very sorry to be this person but the build is failing, mind fixing that? Then feel free to merge :)

**Comment by hola-soy-milk** [comment]
omg don't worry, I needed to fix a few things!

**Comment by hola-soy-milk** [comment]
Ok phew, I think we're good for another review, please :)

**Comment by SaraVieira** [comment]
Danke!!!

**Pull request merged**